### PR TITLE
chore(docker-compose): bump redis version

### PR DESCRIPTION
**Pull request by eran-levy**
bump redis version in docker-compose-non-dev.yml.
solves #14322

### CODE REVIEW

**Review by nytai — APPROVED** [review]

**Review by amitmiran137 — APPROVED** [review]

**Review by villebro — COMMENTED** [review]
LGTM, but pinging @robdiciuccio just to double check - are we ok to bump redis to 6.x, or do we need to stay on 5.x for now?

**Comment by amitmiran137** [comment]
On the helm chart we are using the 6.x

File: docker-compose-non-dev.yml [hunk 1]
```diff
@@ -26,7 +26,7 @@ x-superset-volumes: &superset-volumes
 version: "3.7"
 services:
   redis:
-    image: redis:3.2
+    image: redis:6.2
```
Comment (nytai) [inline docker-compose-non-dev.yml:29]: Let's change this to `:latest` so it's in sync with the [default docker-compose file](https://github.com/apache/superset/blob/master/docker-compose.yml#L32)

Comment (nytai) [reply docker-compose-non-dev.yml:29]: actually, I'm about to make some changes to this file. I'll take care of it

**Comment by villebro** [comment]
> On the helm chart we are using the 6.x

Good to know, I'll bump it in my local devenv too

**Pull request merged**